README.md

```diff
@@ -209,7 +209,7 @@ It "renders" out any `<Text />` values in its child's props.
 
 -   `props` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** props
     -   `props.id` **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Key to look up in intl dictionary, within any parent scopes (`$scope1.$scope2.$id`)
-    -   `props.fields` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** Values to inject into template `{{fields}}` (optional, default `{}`)
+    -   `props.fields` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** Values to inject into template `{{fields}}`.  Values in the fields object will be interpreted as strings, except `<Text />` nodes which will be interpreted/translated (optional, default `{}`)
```

Comment:
The word "interpreted" twice here might be confusing. The second "interpreted/translated" could become "resolved".

Reply:
I like the sound of that.

Reply:
I updated the text with that value in the latest commit